cmd/manager/main.go

```diff
@@ -167,13 +182,53 @@ func main() {
 		log.Info("Created/updated the MemberStatus resource")
 	}()
 
-	log.Info("Starting the Cmd.")
-
 	// Start the Cmd
+	go func() {
+		if err := idlerClientCache.Start(stopChannel); err != nil {
+			log.Error(err, "failed to start idler cache")
+			os.Exit(1)
+		}
+	}()
 	if err := mgr.Start(stopChannel); err != nil {
-		log.Error(err, "Manager exited non-zero")
+		log.Error(err, "Default manager exited non-zero")
 		os.Exit(1)
 	}
+
+	log.Info("Starting the Cmd.")
+}
+
+// newIdlerClient creates a new client for the Idler Controller.
+// As opposed to the client used in other controllers, this one watches resources in all namespaces.
+// But since the Idler controller only cares about Idler resources (cluster-wide) and Pods (namespaced),
+// this client will not store all other namespaced resources (secrets, etc.).
+// This will help keeping a reasonnable memory usage for this operator.
```

Comment:
small typo
```suggestion
// This will help keeping a reasonable memory usage for this operator.
```

Reply:
mind you, it takes 2 `n` in French! https://translate.google.com/#view=home&op=translate&sl=fr&tl=en&text=raisonnable 😂